packages/back-end/src/util/handler.ts

```diff
@@ -147,6 +149,54 @@ export function getBuild() {
   return build;
 }
 
+export async function validateIsSuperUserRequest(req: { userId?: string }) {
+  if (!IS_MULTI_ORG) {
+    throw new Error("This endpoint requires multi-org mode.");
+  }
```

Comment:
Should also check that they have the enterprise feature for multiOrg.  Adnan just merged code today to enable this in REST endpoint, so if you pull in the latest main, it should be easy to add here.

Reply:
So I added: ```  if (req.organization) {
    if (!orgHasPremiumFeature(req.organization, "multi-org")) {
      throw new Error("This endpoint requires an Enterprise plan.");
    }
  }``` Is that what you meant?